### PR TITLE
Create PULL_REQUEST_TEMPLATE.yml

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

**Review by vjik — CHANGES_REQUESTED** [review]
Seems, it's not work. And PR created as usually.

File: .github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Pull Request
+description: Create a pull request to propose and collaborate on changes to this repository.
+labels:
+  - suggestion
```
Comment (samdark) [inline .github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.yml:4]: What does that mean? Where would it be added?

File: .github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+body:
+- type: markdown
+  attributes:
+    value: |
+      > If you have encountered a security related issue with Yii, please use the [security issue form](https://www.yiiframework.com/security) to report any security issues you have encountered to us. ***WARNING!*** DO NOT use the issue tracker or discuss it in the public forum as it will cause more damage than help.
+- type: textarea
+  id: description
+  attributes:
+    label: Please describe your suggestion in detail.
```
Comment (samdark) [inline .github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.yml:13]:
```suggestion
    label: Please describe your pull request in detail.
```

File: .github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.yml (hunk 1 above)
Comment (luizcmarin) [reply .github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.yml:4]: removed

**Pull request closed without merging**